contrib/gcppubsub/pkg/reconciler/gcppubsubsource.go

```diff
@@ -281,6 +299,29 @@ func (r *reconciler) deleteSubscription(ctx context.Context, src *v1alpha1.GcpPu
 	return sub.Delete(ctx)
 }
 
+func (r *reconciler) reconcileEventTypes(ctx context.Context, src *v1alpha1.GcpPubSubSource) error {
+	args := r.newEventTypesReconcilerArgs(src)
+	return r.eventTypeReconciler.ReconcileEventTypes(ctx, src, args)
+}
+
+func (r *reconciler) newEventTypesReconcilerArgs(src *v1alpha1.GcpPubSubSource) *eventtype.ReconcilerArgs {
+	arg := &eventtype.EventTypeArgs{
+		Type: v1alpha1.GcpPubSubSourceEventType,
+		// Using the google cloud project and the topic as source.
+		// This should match what is populated in the adapter.
```

Comment:
If there were a function in the gcppubsub types package producing this string, we wouldn't need this coupling.

Reply:
Done!